extensions/google-translate/src/simple-translate.ts

```diff
@@ -96,3 +96,18 @@ export async function doubleWayTranslate(text: string, options: LanguageCodeSet)
     ]);
   }
 }
+
+export async function multiWayTranslate(text: string, sourceLang: LanguageCode, targetLangs: LanguageCode[]) {
+  if (!text) {
+    return [];
+  }
+
+  return await Promise.all(
+    targetLangs.map((lang) =>
+      simpleTranslate(text, {
+        langFrom: sourceLang,
+        langTo: lang,
+      }),
+    ),
+  );
+}
```

Comment:
I have a suggestion about translations. So as we have ability to reorder target languages, I think we can utilize it. And instead of translating to all the targets in one Promise, we can translate every language in own `List.Item`. And show some loader during translation on each item

So it can looks like this:

```
{targetLangs.map(t => (
  <TranslationListItem sourceLang={sourceLang} targetLang={t} text={text} />
)}
```

Reply:
done! waiting for your feedback. :)


https://github.com/raycast/extensions/assets/45076094/6278f480-23f9-4fb3-8efe-755fe0484ce0

Reply:
Looks awesome!